### PR TITLE
Fix for issue #60

### PR DESCRIPTION
**Pull request by KristianKarl**
I forgot to propagate the earlier fix for where the original problem occurred, namely in the REST call getStatistics.

### CODE REVIEW

**Comment by KristianKarl** [comment]
When merged, Im gonna cherry pick the fix to 3.4.1, and create new 3.4.2 release.

**Comment by nilols** [comment]
Looks like you need to add tests to keep codecov happy =)

File: graphwalker-java/src/main/java/org/graphwalker/java/test/Result.java [hunk 1]
```diff
@@ -201,7 +201,7 @@ public void updateResults(Machine machine, Map<Context, MachineException> failur
       results.put("requirementsFailed", requirementsFailedJson);
     }
 
-    if (failures.size() > 0) {
+    if (failures != null && failures.size() > 0) {
```
Comment (nilols) [inline graphwalker-java/src/main/java/org/graphwalker/java/test/Result.java]: In core we have the Objects helper class that gives you a isNotNullOrEmpty(failures) method, maybe it's better to use that from a code readability point of view

File: graphwalker-restful/src/main/java/org/graphwalker/restful/Restful.java [hunk 2]
```diff
@@ -220,7 +221,9 @@ public String getStatistics() {
     logger.debug("Received getStatistics");
     JSONObject obj = new JSONObject();
```
Comment (nilols) [inline graphwalker-restful/src/main/java/org/graphwalker/restful/Restful.java]: I think we should avoid shortening the variable names

**Comment by KristianKarl** [comment]
@nilols Thanks for the review. I fixed your comment, expect the test coverage. I plan do implement more test coverage for this and some other modules in a separate PR.

**Comment by nilols** [comment]
👍

**Pull request merged**